### PR TITLE
Bed Items

### PR DESCRIPTION
**Pull request by Ayesha-678**
New bed Assets (credit to Gnarp for those!)
Wrapped blanket bindings
Blanket Hood

### CODE REVIEW

**Comment by ace-1331** [comment]
After merging #1336, I'm going through PRs to check what might conflict/be missing.

When you resolve the conflicts, please make sure the items retain the proper SetPose/AllowPose/AllowActivePose, etc. set in 1336 as it appears this PR might change some of the same lines for the bed in particular.

**Comment by Ayesha-678** [comment]
Should be fixed.

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 1]
```diff
@@ -2511,14 +2513,18 @@ var AssetFemale3DCG = [
...
-			{ Name: "Crib", Fetish: ["ABDL"], Priority: 1, Value: 100, Difficulty: 0, SelfBondage: 1, Time: 15, RemoveTime: 10, IsRestraint: true, AllowLock: true, Left: -30, Top: -235, Effect: ["Freeze"], RemoveAtLogin: true, Prerequisite: ["AllFours", "NotSuspended", "NotHogtied", "NoFeetSpreader"], Extended: true, AllowType: ["Open", "Closed", "Stuffed"], SetPose: ["BaseLower"] },
-			{ Name: "Bed", Value: 100, Priority: 1, Difficulty: -20, SelfBondage: 0, Time: 5, RemoveTime: 5, Effect: ["Freeze", "Mounted"], Prerequisite: ["NotKneeling", "AllFours", "NotSuspended", "NotHogtied", "NoFeetSpreader"], BuyGroup: "Bed", RemoveItemOnRemove: [{ Name: "Covers", Group: "ItemAddon" }], SetPose: ["BaseLower"] },
+			{ Name: "DisplayCase", Fetish: ["Metal"], Value: 60, Difficulty: -2, SelfBondage: 1, Time: 15, RemoveTime: 10, AllowLock: true, Prerequisite: ["NotSuspended"], Effect: ["Prone", "Enclose", "DeafLight", "GagLight", "Freeze"], Alpha: [{ Masks: [[1, 1, 70, 999], [420, 1, 80, 999]] }], RemoveAtLogin: true },
```
Comment (ace-1331) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:2516]: Some of these item devices are losing their Audio, SetPose, etc. after the merge

**Review by ace-1331 — CHANGES_REQUESTED** [review]

**Comment by Ayesha-678** [comment]
Should be fixed again, i hope!

File: BondageClub/Assets/Female3DCG/Female3DCG.js [hunk 2]
```diff
@@ -2788,28 +2789,18 @@ var AssetFemale3DCG = [
...
-					{ Name: "CoverOuter", ColorGroup: "Upper" },
-					{ Name: "CoverInner", ColorGroup: "Lower" },
-					{ Name: "TopPillow", ColorGroup: "Upper" },
-					{ Name: "BottomPillow", ColorGroup: "Lower" },
-					{ Name: "Mattress", ColorGroup: "Lower" },
-					{ Name: "Headboard", ColorGroup: "Lower" },
-				]
-				
-			},
+			{ Name: "DisplayCase", Fetish: ["Metal"], Value: 60, Difficulty: -2, SelfBondage: 1, Time: 15, RemoveTime: 10, AllowLock: true, Prerequisite: ["NotSuspended"], Effect: ["Prone", "Enclose", "DeafLight", "GagLight", "Freeze"], Alpha: [{ Masks: [[1, 1, 70, 999], [420, 1, 80, 999]] }], RemoveAtLogin: true },
```
Comment (elliesec) [inline BondageClub/Assets/Female3DCG/Female3DCG.js:2792]: Looks like a bunch of the devices are still missing audio entries

**Review by elliesec — CHANGES_REQUESTED** [review]
Mostly looks good to me! Only two points:
* Audio entries still appear to be missing for some of the devices
* The assets/CSV entries for the blanket hood all appear to be in place, but there doesn't appear to be an entry for it in `Female3DCG.js`

**Comment by Ayesha-678** [comment]
i think i messed this up too badly >.< I'll have to remake it, starting with the new bed assets.

**Pull request closed without merging**